<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SharedResourcesDialog</class>
 <widget class="QWidget" name="SharedResourcesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <property name="whatsThis">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="markAsRemove">
          <property name="toolTip">
           <string extracomment="@info:tooltip">Mark all selected items to be removed from your project</string>
          </property>
          <property name="text">
           <string extracomment="@action:button">Mark as: Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="markAsConvert">
          <property name="toolTip">
           <string extracomment="@info:tooltip">Mark all selected items to be converted to local resources</string>
          </property>
          <property name="text">
           <string extracomment="@action:button">Mark as: Convert</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="markAsKeep">
          <property name="toolTip">
           <string extracomment="info:tooltip">Mark all selected items to be kept as shared resources</string>
          </property>
          <property name="text">
           <string extracomment="@action:button">Mark as: Keep</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeView" name="resourceView">
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="rootIsDecorated">
         <bool>true</bool>
        </property>
        <attribute name="headerDefaultSectionSize">
         <number>300</number>
        </attribute>
        <attribute name="headerHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="headerShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
